openupgradelib/openupgrade.py

```diff
@@ -398,6 +398,27 @@ def drop_columns(cr, column_spec):
                         table, column)
 
 
+def drop_m2m_table(cr, table_spec):
```

Comment:
@hbrunn need this be included into the `_all_` dict? I'm inclined to guess so without being certain about it.